src/Monitors/IMUDownlink.cpp

```diff
@@ -29,7 +29,8 @@ void IMUDownlink::execute()
     }
 
     // need to be stored in sfr later (time to record imu data after deployment)
-    if (millis() - sfr::mission::time_deployed > 60 * constants::time::one_second) {
+    // 60 seconds is about 14 packets
+    if (sfr::mission::deployed && millis() - sfr::mission::time_deployed > 60 * constants::time::one_second) {
         sfr::imu::sample_gyro = false;
         sfr::imu::report_written = true;
```

Comment:
should this be set to false once imu_dlink is emptied in IMUDownlinkReportMonitor.cpp (line 44ish)

Reply:
I don't believe so because report_written just represents if 60 seconds has passed since lightsail deployment. Once this is true, it will never be false again, right?